target/linux/bcm53xx/patches-5.15/180-usb-xhci-add-support-for-performing-fake-doorbell.patch

```diff
@@ -90,33 +90,38 @@ it on BCM4708 family.
  /*
   * Reset a halted HC.
   *
-@@ -611,10 +654,20 @@ static int xhci_init(struct usb_hcd *hcd
+@@ -629,11 +672,24 @@ static int xhci_run_finished(struct xhci
+ 	temp = readl(&xhci->ir_set->irq_pending);
+ 	writel(ER_IRQ_ENABLE(temp), &xhci->ir_set->irq_pending);
  
- static int xhci_run_finished(struct xhci_hcd *xhci)
- {
 -	if (xhci_start(xhci)) {
 -		xhci_halt(xhci);
+-		spin_unlock_irqrestore(&xhci->lock, flags);
 -		return -ENODEV;
++	-if (xhci_start(xhci)) {
++	-	xhci_halt(xhci);
++	-	spin_unlock_irqrestore(&xhci->lock, flags);
++	-	return -ENODEV;
```

Comment:
Perhaps you can copy this patch from `patch-6.1` instead of refreshing it.

Reply:
I did so but the resulting patch needed a manual rebase.  I am finishing the update now and will force push.